src/components/Safety/CampusSafetyHeatmap.tsx

```
@@ -0,0 +1,496 @@
+    const rect = canvas.getBoundingClientRect();
+    const x = e.clientX - rect.left;
+    const y = e.clientY - rect.top;
+
+    // Check if clicked an incident pin
+    for (const report of reports) {
+      const p = projectGeoToPixel(
+        report.latitude,
+        report.longitude,
+        canvas.width,
+        canvas.height
+      );
```

ESLint

[error] Replace `⏎········report.latitude,⏎········report.longitude,⏎········canvas.width,⏎········canvas.height⏎······` with `report.latitude,·report.longitude,·canvas.width,·canvas.height`

(prettier/prettier)